src/wavesurfer.ts

```diff
@@ -307,11 +313,21 @@ class WaveSurfer extends Player<WaveSurferEvents> {
 
           // Set the audio position with a debounce
           clearTimeout(debounce)
+          let debounceTime;
+
+          if (this.isPlaying()) {
+            debounceTime = 0;
+          } else if (this.options.dragToSeek === true) {
+            debounceTime = 200;
+          } else if (typeof this.options.dragToSeek === 'object' && this.options.dragToSeek !== undefined) {
+            debounceTime = this.options.dragToSeek['debounceTime'];
```

Comment:
I don't like accessing by field name but the type system wouldn't have it any other way that I could think of. The rest of the added code tries to ensure that this is safe.